backend/canvas_app_explorer/management/commands/manage_pylti.py

```diff
@@ -0,0 +1,63 @@
+import os
+from datetime import datetime
+
+from Crypto.PublicKey import RSA
+from Crypto.PublicKey.RSA import RsaKey
+from django.core.management.base import BaseCommand, CommandParser
+from django.db.utils import IntegrityError
+
+# https://github.com/dmitry-viskov/pylti1.3/blob/master/pylti1p3/contrib/django/lti1p3_tool_config/models.py
+from pylti1p3.contrib.django.lti1p3_tool_config.models import LtiToolKey, LtiTool
+
+
+class Command(BaseCommand):
+    help = "Generate a private/public key pair and add them to the pylti13 database. This can be called with the same issuer + client ID to update a previous key. The Tool Key will be re-used"
+
+    def add_arguments(self, parser: CommandParser):
+        parser.add_argument('--auth_login_url', dest='auth_login_url', type=str, help="Canvas Login URL",
```

Comment:
should those canvas urls be configurable for test vs production Canvas server urls?

Reply:
Not sure if this is exactly what @zqian is saying, but I was thinking you could just have the user provide the platform URL, and have the endings be configurable via environment variables with good defaults. I think this is fine, but it would make it a lot easier for test configuration if you just had to provide `--platform=canvas.test.instructure.com` and instead of entering all these `_url` flags on the command line.

Reply:
All of these are optionally configurable.

It doesn't **seem** to matter on test/dev/prod which ones you use. You probably should use the one for your server's URL but it works for me even with these defaults, at least on dev. 

If it doesn't work you can add these command line arguments and run it again and it will update it. 

Yeah, maybe rather than making these all individually configurable (or optionally in addition to) we could just have like a platform URL as you suggest. 

Actually that's probably easier for this really and a good idea. I'll just remove all those and if `--platform` is defined we can use that, otherwise just use `canvas.instructure.com`.

Reply:
I think dev uses `canvas.instructure.com` for the platform; not sure if that works for test or beta though.

Reply:
This canvas.instructure.com setting also works for Canvas test server. I tested it with the integration of my localhost.

I am confused now. Is the platform value already remains unchanged as canvas.instructure.com?

Reply:
I haven't done extensive testing but I think the only thing that doesn't work and return the same results is the jwks URL like this https://canvas.instructure.com/api/lti/security/jwks

But we aren't using this anyway in this app so it doesn't matter. **Possibly** also the OAuth URL's but those are in the .env file. 

I think there's a chance that canvas.instructure.com could be down while our instance is still up and break our app. So for prod and test we probably want to use the domain matching the instance we're testing on.

Reply:
I'm simplifying all of this so you only have to pass platform. 

You can also go into the database and edit these too, but I've found it to be a lot easier to run a management command to get this provisioned that editing them in the db, especially for the first one.

Reply:
I updated the command to just take the platform arg